pkg/sanitizer/helpers.go

```diff
@@ -10,36 +10,61 @@ import (
 
 const blocked = "blocked"
 
-type ClusterMap map[string]string
+// ClusterInfo holds the provider, capacity, and capabilities.
+type ClusterInfo struct {
+	Provider     string
+	Capacity     int
```

Comment:
What is capacity? Capacity of what?

Reply:
cluster capacity, how many % of load can be accepted. Will be in documentation

Reply:
I don't think that capacity makes any sense. There can be more jobs that use less resources. Are there any plans to calculate the workload of the cluster based on the resource usage?

Reply:
yes, based on prometheus data. It is needed for the special events if you want to run with degradated load. It was discussed on arch call